src/test/resources/application-test.yml

```diff
@@ -0,0 +1,28 @@
+local:
```

Comment:
I don't understand the content of this file. The `gcp.version` seems unrelated to the task at hand. Same thing for logging & co.

Reply:
It's just the `application.yml` without the `initializr.*` properties. There didn't seem a lot of point fretting about the rest, since they are pretty fast to bind.

Reply:
If we have a dedicated configuration file for a test (that's not a profile) I'd expect:

1. To only contain what is required
2. To not be named like it's a standard file that would be picked up if a `test` profile is active.